src/Microsoft.TestPlatform.Common/Microsoft.TestPlatform.Common.csproj

```diff
@@ -9,6 +9,7 @@
     <TargetFrameworks>netstandard2.0;netstandard1.3;net451</TargetFrameworks>
     <TargetFrameworks Condition=" '$(DotNetBuildFromSource)' == 'true' ">netstandard2.0;netstandard1.3</TargetFrameworks>
     <WarningsAsErrors>true</WarningsAsErrors>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Is this required? I believe there was an orthogonal effort to make all projects on the vstest repo use the latest version here. Not sure if there are any side-effects of making this change. (Would the OSS dev experience need new requirements for instance). Adding in @nohwnd to see if he has any concerns.

Reply:
I've used that to be able to use switch expressions but with recent changes I think its not needed anymore. Trying to remove it

Reply:
Reverted this change